### DISCUSSION TITLE
Private, authenticated nuget-Feed: PRs for packages not created

### DISCUSSION THREAD

**Discussion by Tobias-08** [renovatebot/renovate#10303] [Request Help] [answered]
Hi,

I am not able to get a private, authenticated nuget-feed (Artifactory) to work with renovate. I added hostRules and packageRules in config.js (selfhosted) but the private feed seems to be ignored (nothing in the logs).

```
  hostRules: [
    {
      hostType: 'nuget',
      matchHost: 'https://<our Artifactory>,
      username: '<username>',
      password: '<password>'
    }
  ],
  packageRules: [
    {
      matchDatasources: [ 'nuget' ],
      registryUrls: [
        'https://api.nuget.org/v3/index.json#protocolVersion=3',
        'https://<our Artifactory-Feed-URL>#protocolVersion=3'
      ]
    }
  ],
```

Any ideas?

Thanks!
Tobias

**Comment by viceice** [comment, marked as answer]
Did you run renovate with debug log level? I'm pretty sure there should be some debug messages about your registry.

I'm using custom registries too with nuget, but not artifactory. I'm using proget and baget.

**Reply by Tobias-08** [reply]
debug-logging did the trick. It was not a renovate problem but a connection problem. I would have assumed that these kind of failures are logged in default level anyway...

Thanks!
